src/main/java/kr/codesquad/library/domain/book/Book.java

```diff
@@ -77,4 +86,50 @@ public void returnBook() {
     public void setCategoryToTest(Category category) {
         this.category = category;
     }
+
+    public Long updateBook(BookFormRequest bookFormRequest, Category category, Bookcase bookcase) {
+        this.title = bookFormRequest.getTitle();
+        this.description = bookFormRequest.getDescription();
+        this.author = bookFormRequest.getAuthor();
+        this.publisher = bookFormRequest.getPublisher();
+        this.publicationDate = bookFormRequest.getPublicationDate();
+        this.imageUrl = bookFormRequest.getImageUrl();
+        this.isbn = bookFormRequest.getIsbn();
+        this.category = changeCategory(category);
+        this.bookcase = changeBookcase(bookcase);
+        return this.id;
+    }
+
+    public Category changeCategory(Category category) {
+        if (isDifferentCategory(category)) {
+            this.category.getBooks().add(this);
+            return category;
+        }
+        return category;
+    }
+
+    public Bookcase changeBookcase(Bookcase bookcase) {
+        this.bookcase = bookcase;
+        return bookcase;
+    }
+
+    private boolean isDifferentCategory(Category category) {
+        return this.category.equals(category);
+    }
```

Comment:
메소드명이 조금 헷가리는 메소드명입니다. DifferentCategory 즉, 카테고리가 다르냐 라고 물었는데 return값에선 같을 때 True로 했습니다. 오히려 메소드명을
```suggestion
    private boolean isSameCategory(Category category) {
        return this.category.equals(category);
    }
```

하면 더욱 이해하기 쉬울거 같아요.

Reply:
이게... 처음에 isSameCategory -> isNotSameCategory -> isDifferentCategory 이런 변환이 있었던거 같은데, 제 실수네요. 수정 하겠습니다!

Reply:
이게... 코드를 짜는 과정에서 isSameCategory -> isNotSameCategory -> isDifferentCategory 이런 변화가 있었던거 같은데, 제 실수네요. 수정 하겠습니다!